sous-chef/sous_chef/date/get_due_date.py

```diff
@@ -31,12 +31,13 @@ class MealTime(ExtendedEnum):
 @dataclass
 class DueDatetimeFormatter:
     anchor_day: str
+    week_offset: int = 1
```

Comment:
Why not use config value and no default value?

Reply:
We could remove the default and always specify it if you prefer that. I wouldn't see any problems with that. My slight preference in such situations is to have a default value that reflects the desired standard behavior.

Reply:
put a general config & then get fill these values in the post_init. then =1 not needed as needs to come from yaml...required for most things